src/app/services/task.service.ts

```diff
@@ -87,6 +88,7 @@ export class TaskService {
   private firestore = inject(Firestore);
   private auth = inject(Auth);
 
+  // TODO: Use Vertex AI
```

Comment:
here too... In addition to adding a comment line here, wdyt about also _commenting out_ this code block to make it clear that it's not needed until you decide to start using Vertex AI?